website/content/docs/operators/deployment/schedulers/mesos-local-mac.md

```diff
@@ -0,0 +1,130 @@
+
+### Scheduler
+Heron needs to know where to load the lib to interact with Mesos. Change the config `heron.mesos.native.library.path` in `heron/config/src/yaml/conf/mesos/scheduler.yaml` to the library path of your Mesos. If you installed Mesos through `brew`, the library path should be `/usr/local/Cellar/mesos/your_mesos_version/lib`.
+
+> Mesos only offers interface in C++ library, which is not portable across platforms. So Heron has to ask you for the path of the library, which is able to run in your platform. 
+
+## Compile Heron
+
+Run the command to build and install Heron:
+
+```bash
+bazel run --config=darwin --verbose_failures -- scripts/packages:heron-client-install.sh --user
+```
+This will build everything into `$HOME/.heron`, including all the configurations. In future releases Mesos Scheduler will be included and you will not need to rebuild Heron from source. Then you can add `--config-path=your_conf_path` to your `heron submit` command to tell Heron where to find your config files (default value is `$HOME/.heron/conf`). 
+
+
+## Run Topology in Mesos
+
+After setting up Heron and Mesos, let's submit a topology to test if everything works right, Run:
+
+```bash
+heron submit mesos --verbose ~/.heron/examples/heron-examples.jar com.twitter.heron.examples.ExclamationTopology ExclamationTopology
+```
+
+If the topology is submitted successfully, you will see:
+
+```bash
+[2016-07-25 22:04:41 -0700] com.twitter.heron.scheduler.mesos.MesosLauncher INFO:  For checking the status and logs of the topology, use the working directory $HOME/.herondata/topologies/mesos/$USER/ExclamationTopology
+[2016-07-25 22:04:41 -0700] com.twitter.heron.scheduler.SubmitterMain FINE:  Topology ExclamationTopology submitted successfully
+INFO: Topology 'ExclamationTopology' launched successfully
+INFO: Elapsed time: 4.114s.
+``` 
+
+Note that this doesn't necessarily mean it's successfully launched in Mesos, to check that you can go to the working directory as shown in the output. You will see:
+
+* `heron-examples.jar`: the jar which contains the topology submitted.
+* `heron-conf`: configurations used to launch the topology.
+* `log-files`: directory containing Mesos scheduler's log.
+
+You can look at the log file to see whether the launch succeeded. If it succeeded, at the end of the log file you'll see:
+
+```bash
+[2016-07-25 22:15:47 -0700] com.twitter.heron.scheduler.mesos.framework.MesosFramework INFO:  Received status update [...]
+[2016-07-25 22:15:47 -0700] com.twitter.heron.scheduler.mesos.framework.MesosFramework INFO:  Task with id 'container_1_1469510147073:0' RUNNING
+``` 
+
+If the launch fails, you will see the error message. E.g. if the Heron can't find Mesos library in the given path, it will complain:
```

Comment:
s/you will see the error message/an error message will be included/g

s/E.g. if the Heron can't find Mesos library in the given path, it will complain:/For example, if the Mesos library isn't found in the configured location, the following exception will occur./g